data/exaile.desktop

```diff
@@ -42,3 +42,4 @@ Terminal=false
 StartupWMClass=exaile.py
 MimeType=audio/musepack;application/musepack;application/x-ape;audio/ape;audio/x-ape;audio/x-musepack;application/x-musepack;audio/x-mp3;application/x-id3;audio/mpeg;audio/x-mpeg;audio/x-mpeg-3;audio/mpeg3;audio/mp3;audio/x-m4a;audio/mpc;audio/x-mpc;audio/mp;audio/x-mp;application/ogg;application/x-ogg;audio/vorbis;audio/x-vorbis;audio/ogg;audio/x-ogg;audio/x-flac;application/x-flac;audio/flac;
 Categories=AudioVideo;Audio;Player;GTK;
+Keywords=player,mp3,ogg
```

Comment:
Does someone know better keywords?
It is obligatory to add.
See https://lintian.debian.org/tags/desktop-entry-lacks-keywords-entry and https://wiki.gnome.org/Initiatives/GnomeGoals/DesktopFileKeywords